app/src/main/java/org/p2p/wallet/restore/ui/derivable/DerivableAccountsPresenter.kt

```diff
@@ -41,12 +41,11 @@ class DerivableAccountsPresenter(
     private fun loadSolRate() {
         launch {
             try {
-                val tokenMint = Constants.SOL_MINT
+                val tokenMint = Constants.WRAPPED_SOL_MINT
                 val solRate = tokenServiceRepository.fetchTokenPriceByAddress(
                     networkChain = TokenServiceNetwork.SOLANA,
                     tokenAddress = tokenMint
```

Comment:
I think it should be simplified for clarity (the entire body of `try`)
```suggestion
                tokenServiceRepository.fetchTokenPriceByAddress(
                    networkChain = TokenServiceNetwork.SOLANA,
                    tokenAddress = tokenMint
                )?.also {
                    solRate = it
                    allAccounts = allAccounts.updateWithTotalInUsd(it).toMutableList()
                    filterAccountsByPathAndShow(path)
                }
```